cmd/internal/flowmode/cmd_run.go

```diff
@@ -93,6 +93,8 @@ depending on the nature of the reload error.
 		BoolVar(&r.enablePprof, "server.http.enable-pprof", r.enablePprof, "Enable /debug/pprof profiling endpoints.")
 	cmd.Flags().
 		BoolVar(&r.clusterEnabled, "cluster.enabled", r.clusterEnabled, "Start in clustered mode")
+	cmd.Flags().
```

Comment:
Does it make sense to use this as the hostname? Feels like there could be drift between metric labels and cluster name

Reply:
If the node is unset, this node name defaults to the machine's HOSTNAME.

Is your suggestion that this goes the other way around? That if we set this CLI flag we also override the machine's HOSTNAME?

Reply:
Discussed this and I now understand the use case so a bit of doc on doing this when running multiple nodes on one machine clears this up.